metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_usage_v2.py

```diff
@@ -43,6 +43,20 @@
     "CREATE": OperationTypeClass.CREATE,
     "CREATE_TABLE": OperationTypeClass.CREATE,
     "CREATE_TABLE_AS_SELECT": OperationTypeClass.CREATE,
+    "MERGE": OperationTypeClass.CUSTOM,
+    "COPY": OperationTypeClass.CUSTOM,
+    "TRUNCATE_TABLE": OperationTypeClass.CUSTOM,
```

Comment:
I see that putting `CUSTOM` allows us to show the `customOperationType`, but for other sources we try to label these:

(unity)
```
    QueryStatementType.COPY: OperationTypeClass.INSERT,
    QueryStatementType.UPDATE: OperationTypeClass.UPDATE,
    QueryStatementType.MERGE: OperationTypeClass.UPDATE,
    QueryStatementType.DELETE: OperationTypeClass.DELETE,
    QueryStatementType.TRUNCATE: OperationTypeClass.DELETE,
```

(bigquery)
```
    "MERGE": OperationTypeClass.UPDATE,
```

Reply:
MERGE is in fact UPDATE and/or DELETE and/or INSERT, so I was hesitant to mark it as UPDATE. CUSTOM operationType is not great to use. cc: @jjoyce0510 . We could add more operation types in model maybe ?  I am proposing addition of more operation types to  - MERGE, COPY, TRUNCATE respectively. Does that make sense ?